test-clients/src/main/java/com/hedera/services/bdd/suites/suiterunner/SuiteCategory.java

```diff
@@ -0,0 +1,54 @@
+package com.hedera.services.bdd.suites.suiterunner;
+
+/*-
+ * ‌
+ * Hedera Services Test Clients
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+public enum SuiteCategory {
+	ALL("All"),
+	AUTORENEW_SUITES("Autorenew suites"),
```

Comment:
IMO this `Enum`'s string values should be capitalized and match the identifiers, e.g.: 
```java
public enum SuiteCategory {
    ALL("ALL"),
    AUTORENEW_SUITES("AUTORENEW_SUITES")
    ...
}
```